XenAdmin/Wizards/PatchingWizard/PatchingWizard.cs

```diff
@@ -156,7 +156,7 @@ protected override void UpdateWizardContent(XenTabPage senderPage)
             {
                 var selectedServers = PatchingWizard_SelectServers.SelectedServers;
                 
-                PatchingWizard_PrecheckPage.SelectedServers = selectedServers;
+                PatchingWizard_PrecheckPage.SelectedServers = PatchingWizard_SelectPatchPage.IsInAutomaticMode ? PatchingWizard_SelectServers.SelectedPools.SelectMany(p => p.Connection.Cache.Hosts).ToList() : selectedServers;

```

Comment:
I think it would be nicer to have PatchingWizard_SelectServers.SelectedServers property return the corect list of servers for the AutomaticMode (i.e. all servers in the selected pools)